### PR TITLE
[IAP-3788] - Update information about Try-it button visibility

### PR DESCRIPTION
**Pull request by teivanov**
https://deploy-preview-1540--axway-open-docs.netlify.app/docs/apim_administration/apiportal_admin/customize_APICatalog_overview

Automatically generated by Netlify CMS

### CODE REVIEW

**Review by dginevaAxway — APPROVED** [review]

File: content/en/docs/apim_administration/apiportal_admin/customize_APICatalog_overview.md [hunk 1]
```diff
@@ -76,9 +76,23 @@ You can also choose to use some tags as an internal tool, and hide them from the
...
+   * **No for APIs with the following tags** - Try-it button will be hidden for APIs with specific tags. When this option is selected, a new field is shown - **Hide Inline Try-it for APIs with tags**. In this field you can choose for which tags the Try-it button will be hidden - you can enter multiple tags by separating them by coma. Wildcard symbols (? and *) are also allowed for this field.
+   * **Never** - Try-it button will always be hidden
+4. Click **Save**
+
+
+
+You can also disable the Try-it button for specific types of requests. The Try-It button is enabled for all requests to an API by default. To enable or disable it on specific types of requests (`GET`, `POST`, `PUT`, and so on):
 
 1. In JAI, click **Components > API Portal > API Manager**.
 2. In **Try-It Settings**, enable or disable the Try-It button for each request type.
```
Comment (andreamussap) [inline content/en/docs/apim_administration/apiportal_admin/customize_APICatalog_overview.md:98]: @teivanov  @dginevaAxway  There's an image (Try-It Settings) under this step. I don't think that this image improves the user experience, so I'd like to remove it. What do you think?

Comment (andreamussap) [reply content/en/docs/apim_administration/apiportal_admin/customize_APICatalog_overview.md:98]: I've pushed some changes. Please have a look at everything and let me know if I can approve this PR. Thanks.

**Comment by dginevaAxway** [comment]
@andreamussap  sounds good to me

File: content/en/docs/apim_administration/apiportal_admin/customize_APICatalog_overview.md (hunk 1 above)
Comment (teivanov) [reply content/en/docs/apim_administration/apiportal_admin/customize_APICatalog_overview.md:98]: @andreamussap I'm OK with the changes. Also I don't mind removing the image.

**Review by andreamussap — APPROVED** [review]

**Pull request merged**